server/src/main/java/org/elasticsearch/cluster/SnapshotDeletionsInProgress.java

```diff
@@ -70,6 +70,10 @@ private static boolean assertNoConcurrentDeletionsForSameRepository(List<Entry>
         return true;
     }
 
+    public static SnapshotDeletionsInProgress get(ClusterState currentState) {
```

Comment:
I prefer `state` as the variable name as above. And similar comment for the file below.

Reply:
Sure I can rename that in a sec :)

Reply:
Why not have this method on the cluster ClusterState at first place?

Reply:
I figured I didn't want to blow up the API surface of `ClusterState` further but it's a fair point ... maybe we can just move it there and save even more code. I'll try that out if I can find a second for it.